frontend/src/components/metadata/widgets/MetadataTextField.tsx

```diff
@@ -39,7 +39,7 @@ export const MetadataTextField = (props) => {
 				/>
 			</Grid>
 			<Grid item xs={1} sm={1} md={1} lg={1} xl={1}>
-				{datasetRole.role !== undefined && datasetRole.role !== "viewer" ?
+				{datasetRole !== undefined && datasetRole.role !== "viewer" ?
```

Comment:
you might want to keep the original logic?
`datasetRole !== undefined && datasetRole.role !== undefined && datasetRole.role !== "viewer`

Reply:
We definitely need the whole check as Chen mentioned. else it still shows blank page.

Reply:
It is back in that order.